camel/schemas/openai_converter.py

```diff
@@ -0,0 +1,86 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+
+from typing import Any, Dict, Optional
+
+from pydantic import BaseModel
+
+from camel.models import OpenAIModel
+from camel.types import ModelType
+from camel.utils import (
+    api_keys_required,
+)
+
+from .base import BaseConverter
+
+
+class OpenAISchemaConverter(BaseConverter):
+    def __init__(
+        self,
+        model_type: ModelType = ModelType.GPT_4O_MINI,
+        model_config_dict: Optional[Dict[str, Any]] = None,
+        api_key: Optional[str] = None,
+        target: Optional[BaseModel] = None,
+        prompt: Optional[str] = None,
+    ):
+        """
+        Initializes the OpenAISchemaConverter class
+            with the specified parameters.
+
+        Args:
+            model_type (ModelType): The model type to use.
+            model_config_dict (Optional[Dict[str, Any]]):
+                The model config dict.
+            api_key (Optional[str]): The API key to use.
+            target (Optional[BaseModel]): The target format.
+            prompt (Optional[str]): The prompt to use.
+        """
+        if model_config_dict is None:
+            model_config_dict = {}
+        self.model = OpenAIModel(model_type, model_config_dict, api_key)
```

Comment:
Can we just call the API through the package of OpenAI instead of creating a new Model?